server/db/migrations/20_create_grid_service_instance.rb

```diff
@@ -1,7 +1,12 @@
 class CreateGridServiceInstance < Mongodb::Migration
   def self.up
     GridServiceInstance.create_indexes
-    Container.unscoped.where(container_type: 'container').includes(:grid_service).each do |c|
+    migrate_containers('volume')
+    migrate_containers('container')
```

Comment:
It makes it better, reason: without actually checking volume containers we might miss information during migration. And this might cause service instance to be scheduled to a node that does not have data volume container -> data is lost.